### PR TITLE
Remove extra dns records from seeder

### PR DESCRIPTION
**Pull request by jack60612**
### Purpose:
Remove useless records in the dns server

### CODE REVIEW

File: chia/seeder/dns_server.py [hunk 1]
```diff
@@ -201,10 +201,6 @@ async def dns_response(self, data):
 
             records = {
                 D: IPs,
-                D.ns1: [A(IP)],  # MX and NS records must never point to a CNAME alias (RFC 2181 section 10.3)
-                D.ns2: [A(IP)],
-                D.mail: [A(IP)],
-                D.andrei: [CNAME(D)],
```
Comment (hoffmang9) [inline chia/seeder/dns_server.py:207]: I'm pretty sure these fields may be required to be a well formed DNS nameserver response.

```
dig -6 @2600:1700:22f1:641f:ca10:48f1:7de0:392d chia.hoffmang.com

; <<>> DiG 9.18.12-0ubuntu0.22.04.2-Ubuntu <<>> -6 @2600:1700:22f1:641f:ca10:48f1:7de0:392d chia.hoffmang.com
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 12098
;; flags: qr aa rd ra; QUERY: 1, ANSWER: 32, AUTHORITY: 1, ADDITIONAL: 1

;; QUESTION SECTION:
;chia.hoffmang.com.		IN	A

;; ANSWER SECTION:
chia.hoffmang.com.	300	IN	A	88.76.72.17
chia.hoffmang.com.	300	IN	A	58.186.117.204
chia.hoffmang.com.	300	IN	A	89.216.109.102
chia.hoffmang.com.	300	IN	A	79.131.159.82
chia.hoffmang.com.	300	IN	A	153.227.103.17
chia.hoffmang.com.	300	IN	A	213.45.138.57
chia.hoffmang.com.	300	IN	A	62.11.178.139
...
chia.hoffmang.com.	300	IN	A	177.227.130.8
chia.hoffmang.com.	300	IN	A	5.13.171.194
chia.hoffmang.com.	300	IN	A	187.169.55.26
chia.hoffmang.com.	300	IN	A	117.62.172.191
chia.hoffmang.com.	300	IN	A	171.213.50.71

;; AUTHORITY SECTION:
chia.hoffmang.com.	300	IN	SOA	hoffmang.com. hostmaster.hoffmang.com. 1619105224 10800 10800 604800 1800

;; ADDITIONAL SECTION:
chia.hoffmang.com.	300	IN	NS	hoffmang.com.

;; Query time: 4 msec
;; SERVER: 2600:1700:22f1:641f:ca10:48f1:7de0:392d#53(2600:1700:22f1:641f:ca10:48f1:7de0:392d) (UDP)
;; WHEN: Mon Jul 17 12:40:19 PDT 2023
;; MSG SIZE  rcvd: 608
```

Comment (jack60612) [reply chia/seeder/dns_server.py:207]: I @'ed you on keybase.

Comment (jack60612) [reply chia/seeder/dns_server.py:207]: the ns responses are generated below.

**Review by cmmarslender — APPROVED** [review]
Seems reasonable to me, I don't think we need `mail.<host>`, `ns1.<host>`, or `ns2.<host>` returning `127.0.0.1`

I deployed to an ephemeral network, and was receiving responses as usual with dig.

**Comment by jack60612** [comment]
closing in favor of #15781

**Pull request closed without merging**